.github/workflows/iroha2-all-in-one-publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: iroha2-all-in-one-publish
+
+on:
+  push:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+env:
+  IMAGE_NAME: cactus-iroha2-all-in-one
```

Comment:
@petermetz Please upload this image to main ghcr so I can replace references in the commit :)